src/Microsoft.DotNet.Cli.Utils/CommandResolution/IPlatformCommandSpecFactory.cs

```diff
@@ -0,0 +1,23 @@
+using System;
```

Comment:
I'm not crazy about the use of "Platform" in the name here. These are commands that are being found in the Environment or the Path or the Shell? Platform implies to me that there are differences based on TFM/RID...